R/commonly_used_db_data.R

```diff
@@ -76,6 +77,10 @@ extract_vaccination_history <- function(con, touchstone_cov = "201710gavi", touc
   } else {
     p_int_pop <- external_population_estimates
   }
+
+  if(nrow(p_int_pop) == 0){
+    stop("No population data extracted - you might have specified invalid touchstone_pop.")
```

Comment:
it is useful to add tests to cover this sort of thing to make sure the error cases that you are trying to catch are caught. Now is really the best time to do it as you probably have a good reason for writing that.

Reply:
Thanks @richfitz. I will do as a seperate task. ATM, I am sure this is not an urgent issue comparing with the set of reports pending on vimpact.